File: flyappy_autonomy_code/src/flyappy_autonomy_code_node.cpp
#include <ros/ros.h>

#include "flyappy_autonomy_code/flyappy_ros.hpp"

int main(int argc, char **argv)
{
    ros::init(argc, argv, "flyappy_autonomy_code");
    ros::NodeHandle nh;

    FlyappyRos flyappy_ros{nh};

    ros::spin();

    return 0;
}

File: flyappy_autonomy_code_py/src/flyappy_autonomy_code/flyappy_autonomy_code_node.py
#!/usr/bin/env python3

import rclpy

from flyappy_autonomy_code.flyappy_ros import FlyappyRos


def main() -> None:
    rclpy.init()
    node = rclpy.node.Node('flyappy_autonomy_code_py')
    FlyappyRos(node)
    rclpy.spin(node)


if __name__ == '__main__':
    main()
